spec/features/forgot_password_spec.rb

```diff
@@ -3,6 +3,7 @@
 describe "password retrieval", :js => true do
   context "when a user attempts to retrieve their password" do
     let(:user) { create(:user) }
+    let(:error_message) { I18n.t('passwords.send_instructions') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.